diff --git a/node_modules/react-native-hce/android/build.gradle b/node_modules/react-native-hce/android/build.gradle
index 25c6ac4..d196602 100644
--- a/node_modules/react-native-hce/android/build.gradle
+++ b/node_modules/react-native-hce/android/build.gradle
@@ -4,7 +4,6 @@ buildscript {
 
   repositories {
     google()
-    jcenter()
   }
 
   dependencies {
@@ -52,7 +51,6 @@ android {
 
 repositories {
   mavenCentral()
-  jcenter()
   google()
 
   def found = false
